src/App.vue

```diff
@@ -5,12 +5,12 @@
     <DeviceDetector @DeviceProvided="$setLocaleDevice" :breakpoints="breakpoints" />
     <Tagging />
     <UrlHandler env="env" />
-    <XModal v-if="isOpen" />
+    <XModal v-if="isOpen" :openEvents="openEvents" />
```

Comment:
The only issue I found is this one. I think this XModal should be moved to x-components. And modals, like panels should have its own module.

Reply:
I think there is already a task to create an `XModal` in `x-components` that is going to serve as the main container of the application.